File: src/leetcode/2026/03/Solution709.cpp
#include <iostream>
#include <string>
using namespace std;

class Solution {
public:
    string toLowerCase(string s) {
        for (char &c : s) {
            if (c >= 'A' && c <= 'Z') {
                c = c - 'A' + 'a';
            }
        }
        return s;
    }
};

int main() {
    Solution solution;
    
    string input = "Hello, World!";
    string output = solution.toLowerCase(input);
    cout << "Input: " << input << endl;
    cout << "Output: " << output << endl; // Should print "hello, world!"
}

File: src/leetcode/2026/03/Solution709.java
public class Solution709 {
    public String toLowerCase(String s) {
        return s.toLowerCase();
    }

    public static void main(String[] args) {
        Solution709 solution = new Solution709();

        String input = "Hello";
        String output = solution.toLowerCase(input);
        System.out.println("Output: " + output);

        input = "here";
        output = solution.toLowerCase(input);
        System.out.println("Output: " + output);

        input = "LOVELY";
        output = solution.toLowerCase(input);
        System.out.println("Output: " + output);
    }
}
